pkg/scte35/stream_test.go

```diff
@@ -0,0 +1,107 @@
+package scte35
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func AreEqualJSON(s1, s2 []byte) (bool, error) {
+	var o1 interface{}
+	var o2 interface{}
+
+	var err error
+	err = json.Unmarshal([]byte(s1), &o1)
+	if err != nil {
+		return false, fmt.Errorf("Error mashalling string 1 :: %s", err.Error())
```

Comment:
Prefer lowercase and wrapped errors. e.g. `fmt.Errorf("error marshalling: %w", err)`